### ISSUE TITLE
Add option to output all fields even if empty

### ISSUE THREAD

**Issue by stecydube** [danielo515/obsidian-modal-form#68]
Using it to pre-populate my frontmatter and I want the field names aven if they have no value, so the user can enter them manually later on.

**Comment by danielo515** [comment]
This has been in my mind for some time already. I guess you mean in the two
convenience methods as frontmatterString and as dataview string, right?

**Comment by stecydube** [comment]
Yes those two but I believe this could be useful for `getData()` also.

Maybe also `asString()` as well where one would want to replace an old string with new and same data coming from the existing note.

**Comment by Learwin** [comment]
I recently stumbled upon this plugin and this would be exactly what I'm currently missing. It would be lovely to still create the properties even if no value was input in the template.
